tools/scripts/hostapd.conf

```diff
@@ -9,18 +9,15 @@ ctrl_interface=/var/run/hostapd
 wpa=0
 bridge=br-lan
 ssid=dummy_ssid
-bssid=10:50:F1:20:D8:0E
```

Comment:
Consider keeping this even though it will be selected automatically, to keep the hostapd.conf similar to what we have in Intel reference.